docs/register-bitcoin-stake.md

```diff
@@ -714,8 +757,9 @@ message MsgWithdrawReward {
 
 The message defines the following fields:
 * `type`: Specifies the stakeholder type for reward withdrawal. Allowed values:
-  * `finality_provider`
-  * `btc_staker`
+  * `finality_provider` - Finality provider commission and rewards
```

Comment:
What about finality providers and validators that want to receive costaking rewards?